internal/credsretriever/refreshing_cache.go

```diff
@@ -69,6 +82,15 @@ var (
 )
 
 const (
+	defaultThrottlingKey             = "default-throttling-key"
+	defaultThrottlingMsg             = "account throttling"
+	defaultThrottledRequestCacheSize = 10
+	// default timeout for AWS credential provider is 1 sec:
+	// https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-credential-providers/
```

Comment:
Can we look at the Go SDK for consistency?

Reply:
Actually I looked deep into the http client in Go SDK v2, it is the `DefaultDialKeepAliveTimeout` of 30 secs in `endpointcreds`:

https://github.com/aws/aws-sdk-go-v2/blob/ebb7c024c114fe872f65f8a1d58ba49f0cc1a376/aws/transport/http/client.go#L172-L175